sample/scripts/sample.ts

```diff
@@ -10,6 +11,7 @@ window.onload = () => {
     initEditor();
     initFormatBar();
     initOptions();
+    initSampleCode();
     switchTab('quickstart');
 };
 
```

Comment:
So this change only add one sample code. I'd like to see more functionalities to be completed then check in. If the sample code showing in the UI doesn't reflect any change of user choice, it may mislead users.

Reply:
Added logic to reflect user choice for plugins and format.